### DISCUSSION TITLE
Deploying

### DISCUSSION THREAD

**Discussion by Prexei** [community/community#66084] [Discussions]
### Select Topic Area

Question

### Body

so I have this repository with basic HTML, CSS, JS files. I have chosen main branch and saved it to pages its been 10 hrs and I cant see my website hosted on internet.
github-issue

![image](https://github.com/community/community/assets/134620892/7e9cd4fc-1ac8-4874-9658-5ad0d3d47c39)

**Comment by k-arthik-r** [comment]
If you want to Host your Website using GitHub pages follow these steps:

1. Create a new branch with the name ' gh-pages '.

![image](https://github.com/community/community/assets/111432615/317343f4-20dd-4405-ac41-ad79e07b7ed0)


2. Push a copy of all your files into that branch.

![image](https://github.com/community/community/assets/111432615/9fb7464d-f6d1-4b77-9e27-1dcbe34914ea)

3. Under build and Deployment, select Deploy from branch.
set branch as gh-pages and save it.

![image](https://github.com/community/community/assets/111432615/b77a2b55-67bf-4885-832d-66e9d5218e78)



4. After some times your page will be live at :  

```bash
https://<user_name>.github.io/<Repository_name>/
```

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
